### DISCUSSION TITLE
Alpaca's Mentions

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#238] [Show and tell]
Hi I just found out Alpaca was mentioned in a couple of websites including Forbes, so I wanted to keep track of those mentions here.

I just think it is neat how this 'test app' I was making to learn GTK4 now is getting some recognition

## Flathub Main Page
Jul, 28 2024 | https://github.com/Jeffser/Alpaca/discussions/169
![image](https://github.com/user-attachments/assets/5c5923b1-1596-434d-9d3e-0a335f4b1a42)

## Ubuntu Shell
Aug, 1 2024 | [Ubuntu Shell](https://ubuntushell.com/install-alpaca-on-linux/)
![image](https://github.com/user-attachments/assets/55ab67f4-c165-4617-999a-2fe0e94d36e4)

## Forbes
Aug, 7 2024 | [Forbes](https://www.forbes.com/sites/jasonevangelho/2024/08/07/alpaca-makes-it-easy-to-run-powerful-ai-language-models-on-linux/)
![image](https://github.com/user-attachments/assets/1a2818ba-ee21-4840-a466-d47d4aaebaeb)

## Gnome 47 Wallpaper (Pixels)
Jul, 10 2024 | [Gnome Gitlab](https://gitlab.gnome.org/GNOME/gnome-backgrounds)
![image](https://github.com/user-attachments/assets/2b4fb6db-b97d-4cd1-9874-4965fa593593)

## Flathub's App of the Day
Dec, 24 2024 | [Flathub](https://flathub.org)
![image](https://github.com/user-attachments/assets/59ff9adc-13a0-4f30-bc28-8a3b5450d5b5)

**Comment by loulou64490** [comment]
That's really awesome !

**Comment by olumolu** [comment]
https://youtu.be/JL8sgfGML2A?si=cVsAo27Nz_KVt-w6
Watch after 12:00 min

**Reply by Jeffser** [reply]
oh yeah I saw that, he's one of my favorite influencers, it's truly an honor to know that my code ran in one of his computers

**Reply by loulou64490** [reply]
> oh yeah I saw that, he's one of my favorite influencers, it's truly an honor to know that my code ran in one of his computers

did you know he is french ?? (baguette power :baguette_bread: :muscle: )

**Reply by Jeffser** [reply]
yeah, it's really cool watching a non english speaker making english videos since I'm also not an native english speaker

**Reply by olumolu** [reply]
And the strange part of it is he is even better than many native English.

**Comment by olumolu** [comment]
https://www.reddit.com/r/gnome/comments/1f0ulif/it_has_never_been_easier_to_run_local_ai/
